ev3dev/evb.py

```diff
@@ -0,0 +1,128 @@
+
+"""
+An assortment of classes modeling specific features of the EVB brick.
```

Comment:
EVB _brick_? I would suggest "EVB BeagleBone cape" or whatever those things are called.

Reply:
I'll chop the ` brick` part...I normally here it referred to as just an `EVB`